### PR TITLE
Removed hardcoding from train dashboards:

### PR DESCRIPTION
**Pull request by jkolsen**
Small fixes to remove hardcoding from dashboards.

### CODE REVIEW

**Review by prastut — COMMENTED** [review]
@jkolsen woudn't it be better to use `iterationPerInterface` which is get's in every dashboard file as props `activity.data.iterationPerInterface`

That makes reading code easier to get.

File: ac/ac-train/src/Dashboards/MeanPerTryForEachInterface.jsx [hunk 1]
```diff
@@ -51,7 +51,9 @@ const MeanPerTryForEachInterface = props => {
     }));
 
     const domain =
-      whichDash === 'error' ? { x: [0, 4], y: [0, 1] } : { x: [0, 4] };
+      whichDash === 'error'
+        ? { x: [1, count[interfaces[0]].length], y: [0, 1] }
```
Comment (prastut) [inline ac/ac-train/src/Dashboards/MeanPerTryForEachInterface.jsx]: @jkolsen we can avoid this by using `iterationPerInterface`

**Review by prastut — CHANGES_REQUESTED** [review]

File: ac/ac-train/src/Dashboards/MeanPerTryForEachInterface.jsx (hunk 1 above)
Comment (jkolsen) [reply ac/ac-train/src/Dashboards/MeanPerTryForEachInterface.jsx]: fixed!

**Comment by prastut** [comment]
Thank you @jkolsen!

**Pull request merged**